app/dao/notifications_dao.py

```diff
@@ -459,6 +459,13 @@ def dao_get_notifications_by_to_field(service_id, search_term, statuses=None):
     return results
 
 
+@statsd(namespace="dao")
+def dao_get_notifications_by_reference(references):
```

Comment:
Perhaps we should call this dao_get_notifications_by_references, to indicate we are passing in a list of references.

Reply:
changed in 3594fc.